integration_tests/logger.py

```diff
@@ -0,0 +1,19 @@
+import logging
```

Comment:
This module seems not used in other places?
Currently the other test cases just output stuff to stdout directly through `print`.

Reply:
Yes, this module is not used anywhere, but it is quite helpful in debugging process, as it output logs to a file by using `logger.info()`.
Problem I found with stdout is no output is printed/logged if test passed; and the output is too long if test failed currently.
I can add a small documentation inside the file to explain how to use it.

Reply:
there are several pytest options to control what's output:
- `-s`, don't capture stdout, so all printed stuff is shown
- `--supervisord-quiet`, by default the devnet logs are also redirected to stdout, with this option, they don't, you can still check the log files though, you can find them in `/tm/pytest-xxx`

Reply:
Thank you very much for the tips! 😆  I will remove this file.